agent/bench-scripts/pbench-fio

```diff
@@ -859,15 +866,16 @@ function fio_run_benchmark() {
 								fio_run_job "$iteration" "$benchmark_results_dir" "$fio_job_file" "$dev" "$clients"
 							else
 								# if we are only postprocessing, then we might have to untar an existing result
-								pushd $iteration_dir >/dev/null
-								if [ ! -e sample$sample ]; then
-									if [ -e sample$sample.tar.xz ]; then
+								if [ ! -e $iteration_dir/sample$sample ]; then
```

Comment:
Some sort of tabs vs spaces thing happening here.

Reply:
What sort of thing?

As far as I can discern, the unchanged lines in this file are using hard-tabs for indentation, and the changed lines are doing the same.  (Using hard-tabs wouldn't be my preference, but my IDE handles them nicely, displaying a two-column indentation for them.)  I don't _think_ that I should be reformatting the file for this PR....

Reply:
The "thing" happened a few lines later, and I fixed it.